nx/include/switch/arm/counter.h

```diff
@@ -26,3 +26,11 @@ static inline u64 armGetSystemTickFreq(void) {
     __asm__ ("mrs %x[data], cntfrq_el0" : [data] "=r" (ret));
     return ret;
 }
+
+static inline u64 armNsToTick(u64 ns) {
```

Comment:
armNsToTick/armTickToNs and the changes to event.c should be a separate commit, and this commit should be pushed directly to libnx outside this PR.